docs/formatter.qmd

```diff
@@ -616,5 +616,53 @@ Specifying this would retain the original formatting of the `graph_from_literal(
 skip = ["graph_from_literal"]
 ```
 
-One particularly common use of this is `skip = ["tribble"]`, which you can use to avoid formatting of all `tibble::tribble()` calls.
-In the long term, Air may provide more sophisticated tooling for formatting using a specified alignment.
+# Comment directives
+
+With `# fmt: skip` we've already seen one kind of comment directive, but that's not all.
+Air supports another directive, `# fmt: table`, and we plan to add more in the future.
+
+In general, comment directives are a way to opt out of Air's default automatic formatting and opt into specialized format layouts.
+They generally apply to the expression below the comment, with some possible nuances.
+
+- `skip`: Applies to any following expression.
+- `table`: Applies to the next expression if a function call (e.g. `foo()` or an assigned function call (e.g. `foo <- bar()`)).
+
+# Table formatting
+
+Note: This feature is experimental. We'd be grateful for any feedback!
+
+This directive is particularly useful with calls like tibble's `tribble()` or data.table's `fcase()`.
+It formats calls taking unnamed arguments via `...` in a tabular way.
+
+- Commas are aligned, creating column guides
+- Numeric values are right-aligned, making them easier to read and compare
+- Decimals are aligned, making it easier to distinguish integer and fractional parts
+- Everything else is left-aligned, making it easier to read from left to right
+- Complex calls are formatted in a flat layout, avoiding line breaks where possible.
```

Comment:
```suggestion
- Commas are aligned, creating column guides
- Numeric values are right-aligned
- Everything else is left-aligned
- Decimals are aligned to distinguish integer and fractional parts
- Complex calls are formatted in a flat layout, avoiding line breaks
```

It feels like we can trim this down a good bit and still get the point across

Reply:
hmm